src/docs/01_introduction_and_goals.adoc

```diff
@@ -58,7 +58,24 @@ If you as an architect do not know how the quality of your work will be judged 
 .Form
 A table with quality goals and concrete scenarios, ordered by priorities
 ****
+* _Quality goal_: Annotate and qualify the functionality.
+* _Scenario_: Describe how the system should behave. It specifies quality requirements.
+* _Priority_: It has two values values: importance for success, and how difficult it is to be achieved. 
 
+[options="header",cols="1,2,1"]
+|===
+|Quality goal|Scenario|Priority
+|*_Usability_*  | Non-technical people should easily use the application. | High, medium
+|*_Confidentiality_*  | The control of user data should be decentralized. | High, High
+|*_Security_*  | The application should't have vulnerable points. | High, Medium
+|*_Portability_* | The system could work on a mobile device. | Low, Medium
+|*_Maintainability_* | As Solid is a pioneer evolutive technology, our application should be able to accept changes and be maintained. | Medium, Low
+|*_Performance efficiency_* | The response and processing times of the application should be quick. | High, Medium
+|*_Availability_* | Online independence: The user should be able to use the application without need of Internet connection. | Medium, Medium
```

Comment:
The table should be prioritized, so elements with low importance for success (portability) should go below tables with medium importance (maintainability/availability) and this ones below ones with high importance (performance)